scripts/run-dev.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# ./scripts/run-dev.sh quickly setup cluster for ACK development. It
+# ensures that a K8s cluster is accessible, then configures the ACK CRD, RBAC
+# but not install the controller
+
+set -Eeo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$SCRIPTS_DIR/.."
+
+AWS_SERVICE=$(echo "${AWS_SERVICE:-""}" | tr '[:upper:]' '[:lower:]')
+
+CONTROLLER_NAMESPACE=${CONTROLLER_NAMESPACE:-"ack-system"}
+
+CLUSTER_NAME="ack-dev-cluster-$AWS_SERVICE"
+
+source "$SCRIPTS_DIR/lib/aws.sh"
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/config.sh"
+source "$SCRIPTS_DIR/lib/logging.sh"
+
+source "$SCRIPTS_DIR/controller-setup.sh"
+source "$SCRIPTS_DIR/kind.sh"
+
+ensure_cluster() {
+      info_msg "Creating KIND cluster ..."
+      setup_kind_cluster "$CLUSTER_NAME" "$CONTROLLER_NAMESPACE"
+
+      info_msg "Installing CRDs , common and RBAC manifest..."
+      install_crd_and_rbac "$CONTROLLER_NAMESPACE"
+}
+
+run() {
+    ensure_aws_credentials
+    ensure_cluster
+    local kubeconfig_path="$ROOT_DIR/build/clusters/$CLUSTER_NAME/kubeconfig"
+    info_msg "Before running the controller, you need kubeconfig and aws credentials."
+    info_msg "After executing the above source command to set the kubeconfig to environment, "
+    info_msg "run the following command to start the controller from the controller repo:"
+    echo ""
+    echo "go run ./cmd/controller/main.go --aws-region eu-central-1 --log-level debug --enable-development-logging"
+    echo ""
+    info_msg "if you run the controller from your code editor/IDE, you can set the following environment variables:"
+    echo "KUBECONFIG=$kubeconfig_path"
```

Comment:
This is already called out from inside the `setup_kind_cluster` method. See https://github.com/aws-controllers-k8s/test-infra/blob/main/scripts/kind.sh#L27

Reply:
I think the source command set the kubeconfig in current terminal only,  if starting the controller from goland or vscode, we have to set KUBECONFIG into them. but you're right,  better to mention in document rather than here.